lib/pages/projects/project_bloc.dart

```diff
@@ -37,13 +43,28 @@ class ProjectBloc implements BlocBase {
     });
   }
 
+  void createOrExists(Project project) async {
+    _projectDB.projectExists(project).then((exist) {
```

Comment:
What would have happened if the project already exists? Also, can you please write test for this.

Reply:
Test case `Don't Add project if exist in the project db test` is covering this. (Just created it)